host/containerinit/init.go

```diff
@@ -535,9 +619,6 @@ func Main() {
 		log.Fatalf("Unable to unmarshal config: %v", err)
 	}
 
-	// Propagate the plugin-specific container env variable
-	config.Env["container"] = os.Getenv("container")
-
```

Comment:
It sets an environment variable that should be passed on. It is 'used' in the sense that that env var should be available.